audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/service/KafkaProducerService.java

```diff
@@ -80,6 +80,7 @@ public KafkaProducerService(final Sink<ProducerRecord<String, AuditErrorMessage>
      * @param requests a list of requests
      * @return a {@link ResponseEntity} once all requests have been written to kafka
      */
+    @SuppressWarnings("unused")
```

Comment:
Not sure why this is here.  The method is being used by the method auditRequest in the AuditRestController.

Reply:
The Eclipse IDE throws a waring on `new ProducerRecord<String, AuditSuccessMessage>` at line 141. I have no idea why. Added that in as it's just plain annoying.